components/coding/GoalsSectionComponent.test.ts

```diff
@@ -0,0 +1,64 @@
+import { returnGoalStyle } from "./GoalsSectionComponent";
+
+test("Test returnGoalStyle function - highlighted red", async () => {
+    //Arrange
+    const goal =
+        {"__typename":"user_goals",
+        "createdAt": new Date(2022-11-29),
+        "goalName":"work with Raveen on goals",
+        "goalNotes":null,
+        "id":"a1b08c36-2c8a-49e5-b9c4-f653f826fcee",
+        "updatedAt": new Date(2022-11-29),
+        "userId":"BR9u18SJzvVNzcrkEIcJDPxv1ws1",
+        "isComplete":false,
+        "targetDate": new Date(2022-17-30),
+        "isArchived":null};
+
+  
+    //Act
+    const result = returnGoalStyle(goal);
+    //Assert
+    expect(result).toBe("text-black bg-red-400 rounded-xl p-2");
```

Comment:
I would move this hard coded string into a variable called `cont redHighlightStyle` that will improve readability of this test for someone reading it for the first time.